valkey.conf

```diff
@@ -1308,7 +1308,11 @@ lazyfree-lazy-user-del yes
 # deletion, which can be controlled by passing the [SYNC|ASYNC] flags into the
 # commands. When neither flag is passed, this directive will be used to determine
 # if the data should be deleted asynchronously.
-
+#
+# During a resetting, when a replica performs a CLUSTER RESET, the content
+# of the whole database is removed in order to become an empty primary, this
+# directive will be used to determine f the data should be deleted asynchronously.
+#
```

Comment:
maybe move this section below the original comments so we start with generic information and follow up with specific cases?

```suggestion
#
# When a replica performs a node reset via CLUSTER RESET, the entire
# database content is removed to allow the node to become an empty primary.
# This directive also determines whether the data should be deleted asynchronously.
#
```

Reply:
nice polish, thanks